connectors/data.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "name": "slack",
+        "label": "Slack",
+        "category": "Communication",
+        "description": "Slack is a cloud-based set of proprietary team collaboration tools and services.This connector facilitates automated operations like list channels, list users, send message etc.",
+        "publisher": null,
```

Comment:
publisher?

Reply:
Checked with rest of the solution packs, the publisher in connector's data.json is null, is generated by default on selecting a connector while creating a solution pack